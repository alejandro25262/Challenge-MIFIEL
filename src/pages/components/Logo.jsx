import React from "react";

const Logo = () => {
  return <img alt="Logo" src="images/mifiel_logo.webp" className="w-32" />;
};

export default Logo;
