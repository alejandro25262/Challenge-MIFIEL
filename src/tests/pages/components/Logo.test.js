import { shallow } from "enzyme";
import Logo from "../../../pages/components/Logo";

describe("<Logo />", () => {
  test("match snapshot", () => {
    const wrapper = shallow(<Logo />);
    expect(wrapper).toMatchSnapshot();
  });
});
