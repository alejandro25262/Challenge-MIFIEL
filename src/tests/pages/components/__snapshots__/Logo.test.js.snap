// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> match snapshot 1`] = `
<img
  alt="Logo"
  className="w-32"
  src="images/mifiel_logo.webp"
/>
`;
